frontend/src/component/banners/internalBanners/InternalBanners.tsx

```diff
@@ -11,6 +22,10 @@ export const InternalBanners = () => {
                 .map((banner) => (
                     <Banner key={banner.id} banner={banner} />
                 ))}
+            <ConditionallyRender
+                condition={displayUpgradeEdgeBanner}
+                show={<Banner key={'upgradeEdge'} banner={upgradeEdgeBanner} />}
+            />
```

Comment:
Similarly to https://github.com/Unleash/unleash/pull/5239#discussion_r1413730809 - I would recommend we create a new `EdgeUpgradeBanner` (or similar) component and add it directly to `App.tsx`. I don't think it makes much sense to add it here, especially as it is not an [internal banner](https://docs.getunleash.io/reference/banners).

Reply:
Agreed. Will fix